### PR TITLE
Fix compilation with zookeeper 3.5.5

### PR DESCRIPTION
**Pull request by ijuma**
And whitelist 4lw commands in tests (necessary since ZK 3.5.3).

### CODE REVIEW

**Comment by ijuma** [comment]
This should be merged at the same time as https://github.com/confluentinc/common/pull/211.

File: utility-belt/src/test/java/io/confluent/admin/utils/EmbeddedZookeeperEnsemble.java [hunk 1]
```diff
@@ -119,6 +122,7 @@ public void start() throws IOException {
       s.start();
     }
 
+    System.out.println("Checking ports " + hostPort);
```
Comment (ewencp) [inline utility-belt/src/test/java/io/confluent/admin/utils/EmbeddedZookeeperEnsemble.java]: is this just leftover or is there a reason we're mixing slf4j logs and `System.out.println`?

Comment (ijuma) [reply utility-belt/src/test/java/io/confluent/admin/utils/EmbeddedZookeeperEnsemble.java]: Yes, my bad.

**Comment by ijuma** [comment]
https://github.com/confluentinc/common/pull/211 has been merged, so merging this too. See that PR for details on the cp system tests that were run for verification of this change.

**Pull request merged**